ublox_derive/src/output/gen_recv_code.rs

```diff
@@ -40,7 +40,7 @@ pub fn generate_recv_code_for_packet(dbg_ctx: DebugContext, pack_descr: &PackDes
     quote! {
         #[doc = #struct_comment]
         #[doc = "Contains a reference to an underlying buffer, contains accessor methods to retrieve data."]
-        pub struct #ref_name<'a>(&'a [u8]);
+        pub struct #ref_name<'a>(pub(super) &'a [u8]);
```

Comment:
It's fine but I'd probably go with `pub(crate)` to avoid ever having to understand all of these lines in the future when someone decides to move things around again.

Reply:
Good point indeed!